README.rst

```diff
@@ -95,49 +65,133 @@ Launching a container
 ---------------------
 
 * Once the image is created, you may start a new Docker container using it.
-  In the following example, we will run a container named `sftpplus-instance`,
-  which is using the `sftpplus:3.44.0` image, and which makes all the ports
-  available to the outside world. There are a lot of services and ports::
-
-    docker run -d --name sftpplus-instance \
-        -p 10020:10020 \
-        -p 10080:10080 \
-        -p 10443:10443 \
-        -p 10022:10022 \
-        -p 10023:10023 \
-        -p 10021:10021 \
-        -p 10990:10990 \
-        -p 10900-10910:10900-10910 \
+  In the following example we run a container named ``sftpplus-trial-instance``
+  using the ``sftpplus:3.44.0.trial`` image which publishes all its services
+  to the outside world. There are a lot of services and ports open by default::
+
+    docker run --detach --name sftpplus-trial-instance \
+        --publish 10020:10020 \
+        --publish 10080:10080 \
+        --publish 10443:10443 \
+        --publish 10022:10022 \
+        --publish 10023:10023 \
+        --publish 10021:10021 \
+        --publish 10990:10990 \
+        --publish 10900-10910:10900-10910 \
         sftpplus:3.44.0.trial
 
 * You can check that the container is started with::
 
-    docker ps -a
+    docker ps --all
 
 * And check the logs with::
 
-    docker logs sftpplus-instance
+    docker logs sftpplus-trial-instance
 
-* And stop the container with::
+* To inspect a container which is already running::
 
-    docker stop sftpplus-instance
+    docker exec --interactive --tty sftpplus-trial-instance /bin/sh
 
-* To check how the image is created, you can start with a shell to inspect it::
+* You can stop the container with::
 
-    docker run -it --name sftpplus-debug sftpplus:3.44.0 /bin/sh
+    docker stop sftpplus-trial-instance
 
-* To inspect a container which is already running::
+* And then remove it with::
+
+    docker rm sftpplus-trial-instance
+
+* To remove the trial image altogether::
+
+    docker rmi sftpplus:3.44.0.trial
+
+
+Image Customization
+-------------------
+
+Since the default SSH keys and SSL certificates are automatically generated,
+the default ``Dockerfile`` presented here is not suitable for production.
+
+For production usage you should add your own SSH keys and SSL certificates to
+the ``configuration/`` subdirectory.
+Then modify ``server.ini`` to use these files.
```

Comment:
This is ok.
Not  sure if we should mention that they can also include the content of the certs inside the server.ini

Reply:
Sure, no problem.